src/test/java/org/example/tests/api/spring/user/DELETEUsersTest.java

```diff
@@ -0,0 +1,93 @@
+package org.example.tests.api.spring.user;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.example.tests.api.spring.ApiBaseClass;
+import org.testng.annotations.Test;
+
+import static org.apache.http.HttpStatus.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class DELETEUsersTest extends ApiBaseClass {
+
+    @Test
+    public void deleteUser() {
+
+        Response response = RestAssured.given()
+                .header("app-id", properties.getAppId())
+                .contentType(ContentType.JSON)
+                .delete("/user/63d7914f10fee17a516c598d");
+
+        logResponse(response);
+
+        //Validate user is deleted successfully
+        JsonPath path = response.body().jsonPath();
+        assertNull(path.get("firstName"));
+
+        // Validate status code
+        int statusCode = response.getStatusCode();
+        assertEquals(statusCode, SC_OK);
```

Comment:
In order for this test to be valid you need first to be sure that the user with id 63d7914f10fee17a516c598d exists. This test will fail if you run it several times. If you remember from my first presentation "Ensure your test case has a long life" :)